clang/include/clang/CIR/Dialect/IR/CIRAttrs.td

```diff
@@ -146,6 +146,37 @@ def ConstArrayAttr : CIR_Attr<"ConstArray", "const_array", [TypedAttrInterface]>
   }];
 }
 
+//===----------------------------------------------------------------------===//
+// ConstVectorAttr
+//===----------------------------------------------------------------------===//
+
+def ConstVectorAttr : CIR_Attr<"ConstVector", "const_vector",
+                               [TypedAttrInterface]> {
+  let summary = "A constant vector from ArrayAttr";
+  let description = [{
+    A CIR vector attribute is an array of literals of the specified attribute
+    types.
+  }];
+
+  let parameters = (ins AttributeSelfTypeParameter<"">:$type,
+		        "Attribute":$elts);
```

Comment:
I wrote that just long enough ago that I can't remember if `Attribute` instead of `ArrayAttr` was intentional or not.  I'll try changing it to `ArrayAttr` and see what happens.

Reply:
Done.  Changing the elements field to `ArrayAttr` works fine,